book/modules.md

```diff
@@ -197,8 +197,7 @@ Submodules are modules inside modules. They are automatically created when you c
 
 The difference is that `export module some-module` _only_ adds the module as a submodule, while `export use some-module` _also_ re-exports the submodule's definitions. Since definitions of submodules are available when importing from a module, `export use some-module` is typically redundant, unless you want to re-export its definitions without the namespace prefix.
 
-> **Note**
-> `module` without `export` defines only a local module, it does not export a submodule.
+> **Note** > `module` without `export` defines only a local module, it does not export a submodule.
```

Comment:
this looks like it will break the note, right? :thinking:

Reply:
definitely. fixing it now.

Reply:
Ok, something is not right here. I now have triple-checked and this is not the editor's fault but a git hook messing with Markdown.